packages/graphql/src/classes/utils/asserts-indexes-and-constraints.ts

```diff
@@ -262,55 +262,55 @@ async function getMissingConstraints({
     schemaModel: Neo4jGraphQLSchemaModel;
     session: Session;
 }): Promise<MissingConstraint[]> {
-    const existingConstraints: Record<string, string[]> = {};
-
-    const constraintsCypher = "SHOW UNIQUE CONSTRAINTS";
-    debug(`About to execute Cypher: ${constraintsCypher}`);
-    const constraintsResult = await session.run<{ labelsOrTypes: [string]; properties: [string] }>(constraintsCypher);
-
-    constraintsResult.records
+    const CYPHER_SHOW_UNIQUE_CONSTRAINTS = "SHOW UNIQUE CONSTRAINTS";
+    debug(`About to execute Cypher: ${CYPHER_SHOW_UNIQUE_CONSTRAINTS}`);
+    const constraintsQueryResult = await session.run<{ labelsOrTypes: [string]; properties: [string]; name: string }>(
+        CYPHER_SHOW_UNIQUE_CONSTRAINTS
+    );
+    // Map that holds as key the label name and as value an object with properties -> constraint name
+    const existingConstraints: Map<string, Map<string, string>> = constraintsQueryResult.records
```

Comment:
This function is getting quite big, any change of moving the existing constraints generation to a new function? `getExistingConstraints`?

Reply:
Sounds good, let me separate this method, I tried to limited the refactor here but there is no reason why we cannot improve it a little bit more!